DSA_Session5/src/employee/LinkedList.java

```diff
@@ -0,0 +1,69 @@
+package employee;
+/**
+ * program to get employee list based on their salary
+ * 
+ * @author Aman Gautam
+ * 
+ * Dated -06/08/2019 
+ */
+public class LinkedList {
+	Node headNode;
+
+	/**
+	 * method to insert data into linked list
+	 * 
+	 * @param list {LinkedList} contain employee's data
+	 * @param data of employee
+	 * @return list{List} of employee
+	 */
+	public LinkedList insert(LinkedList list, Employee data) {
```

Comment:
No need LinkedList list, argument. Please take only employee data as an input parameter. And declare some variable in the same class and implement accordingly.

Reply:
extra arguments removed .